QuoteEngine/README.md

```diff
@@ -1,30 +1,177 @@
-Quick start, because I can't be arsed writing a full one ;)
+**Hello!**
 
-SQL setup
----------
 
-1. Create a user and database in your mysql server for the bot to use
-2. Populate the database using the .sql script
+Steps you need to do (or packages you should have) before downloading the script:
+1. Install all necessary packages for eggdrop to make use of mysql (also packages which you need to have are included).
+   <br>
+   For example for ubuntu 18.04 these would be the packages: tcl, tcl-dev, tcllib, tcl-tls, zlibc, zlib1g, tcl-trf, mysqltcl
+2. Install mysql-server and phpmyadmin (phpmyadmin is not really needed, but it helps to manage databases).
+   <br>
+   For example for ubuntu 18.04 these would be the packages: mysql-server, mysql-client, phpmyadmin
+      <br>
+      For phpmyadmin you need the following packages: apache2, php <- install those first. :)
+3. Install eggdrop. Check out here: https://www.eggheads.org/downloads
 
-Note: Give the SQL user minimum permissions, restrict it to just the host the bot runs on, etc.
+In case you have a running bot, you can continue.
 
-TCL setup
----------
 
-1. Install mysqltcl package
-2. Put the QuoteEngine.tcl file in your eggdrop's scripts/ directory
-3. Edit the QuoteEngine-settings.sample.tcl file and save it as
-   QuoteEngine-settings.tcl in your scripts/ directory
-4. Put `source scripts/QuoteEngine.tcl` in your bot's config file
+Database setup, first steps
+---------------------------------------------
+
+1. Create admin user for mysql if you don't have it yet.
+   
+   // In case you have an admin user for your mysql server other than the "root" user, you can skip this. //
+
+   With root user enter the following command in your shell:
+   <br>
+   ```# mysql -u root```
+      <br>
+      This will let you to login to mysql, so you can make changes on/in it.
+ 
+   Create a new admin user (so you can login to phpmyadmin later on):
+   <br>
+   ```mysql> CREATE USER '<user>'@'%' IDENTIFIED BY '<pwhere>';```
+      <br>
+      This will create the <user> with the given password.
+      
+   Grant all right to the user:
+   <br>
+   ```mysql> GRANT ALL PRIVILEGES ON *.* TO '<user>'@'%' WITH GRANT OPTION;```
```

Comment:
As per note above, this user is too powerful and can log in from anywhere :)

Reply:
Update: ok, I see you're adding an admin user but I think it would still be better scoped to @localhost

Reply:
yes. made it like that. if the user wants to use phpmyadmin, probably wont log from localhost. she/he will access via a browser. from local computer. i mean that is what i would do.
(regular users wont install browser on a server and use some X-forwarding tool to get the browser, or wont tunnel for sure)
---
although made a note to disable phpmyadmin after finishing.